doc/ug/dv_methodology/index.md

```diff
@@ -299,60 +299,112 @@ Once the regression tool is developed, it will provide a way to enable these cap
 
 ## Coverage Collection
 
-Collecting, analyzing and reporting coverage with waivers is another requirement to assert 'verification complete'.
+Collecting, analyzing, and reporting coverage with waivers is a requirement to assert 'verification complet'.
```

Comment:
Sorry I didn't notice this last night: "complete" has lost its trailing "e".

Reply:
wierd.. but fixed.. thank you